sdk/src/main/java/com/mapbox/maps/MapboxMapExt.kt

```diff
@@ -0,0 +1,19 @@
+@file:JvmName("MapboxMapUtils")
+
+package com.mapbox.maps
+
+import com.mapbox.geojson.Point
+import com.mapbox.maps.plugin.delegates.MapCameraManagerDelegate
+
+/**
+ * Calculates geographical coordinates(i.e., longitude-latitude pair) that corresponds
+ * to screen box.
+ *
+ * @param screenBox A box on the screen in \link MapOptions#size platform pixels \endlink.
+ *
+ * @return Returns a batch of geographical coordinates corresponding to the screen coordinates
+ * on the screen.
+ */
+fun MapCameraManagerDelegate.coordinatesForPixels(screenBox: ScreenBox): List<Point> {
+  return this.coordinatesForPixels(listOf(screenBox.min, screenBox.max))
```

Comment:
is it important to get real `min` and `max` values? Because right now `ScreenBox` stores `ScreenCoordinate` as first and second argument.

Reply:
if user set in another order. is it critical for internal logic or not?

Reply:
Then that would be a different ScreenBox, right?

Reply:
I mean setting `this.coordinatesForPixels(listOf((1,2), (4,5)))` and `this.coordinatesForPixels(listOf((4,5), (1,2)))` will lead to different results.